Documentation/articles/getting_started/toc.yml

```diff
@@ -1,14 +1,24 @@
+- name: Welcome
+  href: 0_welcome.md
+- name: Setting up your Development Environment
+  items:
+  - name: Windows
+    href: 1_setting_up_your_development_environment_windows.md

```

Comment:
This page appears to be missing in the PR??

Reply:
It's not written, I asked on discord who wants to write it.

Reply:
Me pls sir, can I have some more :D